### PR TITLE
Post training examples

### PR DESCRIPTION
**Pull request by dmahan93**
- recreate zephyr 7b beta from the sft https://huggingface.co/HuggingFaceH4/mistral-7b-sft-beta
- llama 3 configs for sft/dpo/rm on ultrafeedback
- implementation improvements that came up when poking at it (cross entropy is negative logprobs, and we have an efficient cross entropy! no need to do a full gather over the logits 🚀)

### CODE REVIEW

**Comment by Quentin-Anthony** [comment]
> will need to edit the configs once whatever changes in #1276 get finalized

It's been merged. You're clear to do a final pass + test run.

**Review by Quentin-Anthony — APPROVED** [review]

**Comment by Quentin-Anthony** [comment]
Tested and working for me
